irispy/spectrograph.py

```diff
@@ -468,6 +468,8 @@ def read_iris_spectrograph_level2_fits(filenames, spectral_windows=None):
     """
     if type(filenames) is str:
         filenames = [filenames]
+    if not filenames:
```

Comment:
Suggest these two lines be replaced with something like:
```python
if isinstance(filenames, list):
    raise TypeError("filenames must be a string or list of strings giving the filenames to read.")
if len(filenames) == 0:
    raise ValueError("No filenames provided.")
```

Reply:
Ok. Replace them. I kept the "if not filenames" as this is the recommended by PEP8 for checking for empty lists.

Reply:
Ok, I replaced the lines according to your suggestion. But I kept the `if not filenames` form, as this is recommended by PEP for checking empty lists.